lib/timezone/lookup/geonames.rb

```diff
@@ -34,13 +34,37 @@ def lookup(lat, long)
         raise(Timezone::Error::GeoNames, e.message)
       end
 
+      def lookup_with_etc(lat, long)
+        response = client.get(url(lat, long))
+
+        return unless response.body
+
+        data = JSON.parse(response.body)
+
+        return data['timezoneId'] if data['timezoneId']
+
+        return get_etc(data['gmtOffset']) if data['gmtOffset']
+
+        return unless data['status']
+
+        raise(Timezone::Error::GeoNames, data['status']['message'])
+      rescue => e
+        raise(Timezone::Error::GeoNames, e.message)
+      end
+
       private
 
+      def get_etc(offset)
+        return 'Etc/UTC' if offset == 0
+        "Etc/GMT#{format('%+d', offset.to_i)}"
```

Comment:
Well, now for coordinates in the ocean, for example, lookup will rise exception, because both geonames and google do not give timezoneId for such points.

I thought that about including it in the lookup function, but I was not sure about compatibility issues. After all someone may use the fact thel lookup(0,0) raises exception, and when method's behaviour changes, it may couse some problems for people who use this gem.

Maybe it would be better to use it as an option for lookup, like this:
`
def lookup(lat, long, options = {})
...
  return get_etc(data['gmtOffset']) if options[:with_etc] && data['gmtOffset']
...
end  
`

I'll refactor it to include into lookup method, but should I do it optional to not change current behaviour for compatibility?